redbot/core/core_commands.py

```diff
@@ -1922,11 +1922,11 @@ async def allowlist_clear(self, ctx: commands.Context):
         await self.bot._whiteblacklist_cache.clear_whitelist()
         await ctx.send(_("Allowlist has been cleared."))
 
-    @commands.group(aliases=["blacklist", "denylist"])
+    @commands.group(aliases=["blacklist", "list"])
```

Comment:
Did you mean to do this here and in line 1994?

Reply:
Close but not quite. One sec